pkgs/development/ocaml-modules/github/unix.nix

```diff
@@ -12,6 +13,14 @@ buildDunePackage {
   pname = "github-unix";
   inherit (github) version src;
 
+  patches = [
+    # Compatibility with yojson 3.0
+    (fetchpatch {
+      url = "https://github.com/mirage/ocaml-github/commit/487d7d413363921a8ffbb941610c2f71c811add8.patch";
+      hash = "sha256-ThCsWRQKmlRg7rk8tlorsO87v8RWnBvocHDvgg/WWMA=";
+    })
+  ];
+
```

Comment:
Can be dropped, if you bump the version here: https://github.com/mirage/ocaml-github/pull/281